browser-test/src/admin/admin_program_migration.test.ts

```diff
@@ -233,6 +284,13 @@ test.describe('program migration', () => {
       await expect(
         page.getByRole('heading', {name: 'file upload'}),
       ).toBeVisible()
+
+      // Assert the warning about duplicate question names is shown
+      await adminProgramMigration.expectAlert(
```

Comment:
I opted not to generate screenshots for these to avoid clutter, but let me know if screenshots would be helpful.